dropshot/tests/fail/bad_server_endpoint26.stderr

```diff
@@ -0,0 +1,7 @@
+error: endpoint `bad_endpoint` has invalid attributes: missing field `path`
+  --> tests/fail/bad_server_endpoint26.rs:12:1
+   |
+12 | #[dropshot::server]
+   | ^^^^^^^^^^^^^^^^^^^
+   |
+   = note: this error originates in the attribute macro `dropshot::server` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Comment:
This is addressed by https://github.com/oxidecomputer/serde_tokenstream/pull/194.